src/shared/interfaces/Organization.ts

```diff
@@ -0,0 +1,7 @@
+export interface Organisation {
+    name: string;
+    createdEvents: string[],
```

Comment:
Here finally is it going to be an array of strings or array of objects as we've discussed? And isn't it better to be optional?